content/en/docs/tasks/cluster-administration/node-overprovisioning/node-overprovisioning.md

```diff
@@ -35,4 +33,104 @@ metadata:
 value: 1000
 globalDefault: false
 description: "Low priority for placeholder pods to enable overprovisioning."
-```
\ No newline at end of file
+```
+
+Then apply the Priority Class:
+
+```shell
+kubectl apply -f low-priority-class.yaml
+```
+
+Now, to create the Placeholder Deployment, define a deployment that uses the low-priority Priority Class and runs a minimal container.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: placeholder-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: placeholder
+  template:
+    metadata:
+      labels:
+        app: placeholder
+    spec:
+      priorityClassName: low-priority
+      containers:
+      - name: pause
+        image: k8s.gcr.io/pause:3.2
```

Comment:
Not sure if we want to use this image (Google container registry) and this tag (3.2 is very old).

Reply:
I'd use a newer one.